share/mrdocs/addons/generator/adoc/layouts/wrapper.adoc.hbs

```diff
@@ -9,7 +9,7 @@
 
     Breadcrumbs are included in the symbol template as an alternative.
 }}
-= {{> symbol/qualified-name symbol }}
+= {{> symbol/enclosing-namespace symbol }}{{> symbol/qualified-name symbol }}
```

Comment:
This worsens things because we have removed the links to other enclosing namespaces, which the `qualified-name` partial does. It's removing a feature rather than including one.

Reply:
I was surprised the test files didn't change. I think it's because tests bypass the wrapper. But losing this information about the enclosing namespaces in real documentation like Boost.URL sure looks bad.